includes/class-wc-tracker.php

```diff
@@ -540,6 +620,49 @@ private static function get_admin_user_agents() {
 		return array_filter( (array) get_option( 'woocommerce_tracker_ua', array() ) );
 	}
 
+	/**
+	 * Get order counts by payment method
+	 *
+	 * @return array
+	 */
+	public static function get_order_counts_by_payment_method() {
```

Comment:
I think this method can be removed now, right, since [the code above](https://github.com/woocommerce/woocommerce/pull/28584/files#diff-81c4e5a312109472fa8526612aeb12807759e0ee28d912a7dc775c890a909d16R417-R428) calculates the same thing?

Reply:
Yes, as I mentioned [here](https://github.com/woocommerce/woocommerce/pull/28584#issuecomment-749867388), what would you prefer that I do with the order such functions that this code replaces?

Reply:
Ah, I missed that part, sorry. Since they are public, we can't remove them. I think deprecation is probably the right way to go. Thanks for switching to CRUD!